eq-author/src/App/history/HistoryPage.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import styled from "styled-components";
+import { useQuery } from "@apollo/react-hooks";
+
+import CustomPropTypes from "custom-prop-types";
+
+import Error from "components/Error";
+import Loading from "components/Loading";
+
+import HistoryItem from "./HistoryItem";
+import Header from "components/EditorLayout/Header";
+import { Grid } from "components/Grid";
+import questionnaireHistoryQuery from "./questionnaireHistory.graphql";
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`;
+
+const StyledGrid = styled(Grid)`
+  display: flex;
+  flex-direction: column;
+  padding: 1em;
+  border: 1px solid #d6d8da;
+  margin: 0.5em 1em;
+  overflow: hidden;
+  max-width: 60em;
+`;
+
+export const UnwrappedHistoryPageContent = ({ match }) => {
```

Comment:
Any reason behind calling this `UnwrappedHistoryPageContent` as it doesn't look like there's a WrappedHistoryPageContent anywhere?